hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/flink/FlinkCleaner.java

```diff
@@ -49,13 +49,19 @@
   private DistributedFsService dfs;
   
   @Schedule(persistent = false,
-    minute = "0",
-    hour = "1")
+    minute = "*",
+    hour = "*", second = "*")
```

Comment:
Why did you change the frequency?

Reply:
Woopsie was using it for testing, should not have been committed